concepticondata/conceptlists/Pereira-2018-180.tsv

```diff
@@ -0,0 +1,181 @@
+ID	NUMBER	ENGLISH	POSKey	UniversalPos	CONCEPTICON_ID	CONCEPTICON_GLOSS
+Pereira-2018-180-1	1	ability	1	noun	2527	ABILITY
+Pereira-2018-180-2	2	accomplished	3	adj		
+Pereira-2018-180-3	3	angry	3	adj	1826	ANGRY
+Pereira-2018-180-4	4	apartment	1	noun		
+Pereira-2018-180-5	5	applause	1	noun		
+Pereira-2018-180-6	6	argument	1	noun		
+Pereira-2018-180-7	7	argumentatively	4	adv		
+Pereira-2018-180-8	8	art	1	noun		
+Pereira-2018-180-9	9	attitude	1	noun		
+Pereira-2018-180-10	10	bag	1	noun	2468	BAG
+Pereira-2018-180-11	11	ball	1	noun	1845	BALL
+Pereira-2018-180-12	12	bar	1	noun		
+Pereira-2018-180-13	13	bear	1	noun	1820	BEAR
+Pereira-2018-180-14	14	beat	1	noun		
+Pereira-2018-180-15	15	bed	1	noun	1663	BED
+Pereira-2018-180-16	16	beer	1	noun	1639	BEER
+Pereira-2018-180-17	17	big	1	noun	1202	BIG
+Pereira-2018-180-18	18	bird	1	noun	937	BIRD
+Pereira-2018-180-19	19	blood	1	noun	946	BLOOD
+Pereira-2018-180-20	20	body	1	noun	1480	BODY
+Pereira-2018-180-21	21	brain	1	noun	1583	BRAIN
+Pereira-2018-180-22	22	broken	3	adj	588	BROKEN
+Pereira-2018-180-23	23	building	1	noun	3396	BUILDING
+Pereira-2018-180-24	24	burn	2	verb	141	BURN (SOMETHING)
+Pereira-2018-180-25	25	business	1	noun	2931	BUSINESS
+Pereira-2018-180-26	26	camera	1	noun	2618	CAMERA
+Pereira-2018-180-27	27	carefully	4	adv		
+Pereira-2018-180-28	28	challenge	1	noun		
+Pereira-2018-180-29	29	charity	1	noun		
+Pereira-2018-180-30	30	charming	3	adj		
+Pereira-2018-180-31	31	clothes	1	noun	1895	CLOTHES
+Pereira-2018-180-32	32	cockroach	1	noun	1575	COCKROACH
+Pereira-2018-180-33	33	code	1	noun		
+Pereira-2018-180-34	34	collection	1	noun		
+Pereira-2018-180-35	35	computer	1	noun	3086	COMPUTER
+Pereira-2018-180-36	36	construction	1	noun		
+Pereira-2018-180-37	37	cook	2	verb	1100	COOK (SOMETHING)
+Pereira-2018-180-38	38	counting	2	verb	1420	COUNT
+Pereira-2018-180-39	39	crazy	3	adj	1899	MAD
+Pereira-2018-180-40	40	damage	1	noun	2947	DAMAGE (INJURY)
+Pereira-2018-180-41	41	dance	1	noun	3472	DANCE (MOVEMENT)
+Pereira-2018-180-42	42	dangerous	3	adj	3466	DANGEROUS
+Pereira-2018-180-43	43	deceive	2	verb	3230	DECEIVE
+Pereira-2018-180-44	44	dedication	1	noun		
+Pereira-2018-180-45	45	deliberately	4	adv		
+Pereira-2018-180-46	46	delivery	1	noun		
+Pereira-2018-180-47	47	dessert	1	noun		
```

Comment:
OMG our concepticon lacks a concept for `dessert`. That disqualifies us for any merchandising enterprises with Nestlé! Sorry, just find it interesting to note how much what is "basic" seems to have changed. Ignore this comment.